wfl/generate/md/__init__.py

```diff
@@ -43,7 +46,7 @@ def _sample_autopara_wrappable(atoms, calculator, steps, dt, temperature=None, t
         - [ {'T_i': float, 'T_f' : float, 'traj_frac' : flot, 'n_stages': int=10}, ... ] list of stages, each one a ramp, with
         duration defined as fraction of total number of steps
     temperature_tau: float, default None
-        time scale that enables Berendsen constant T temperature rescaling (fs)
+        time scale that enables Berendsen constant T temperature rescaling (fs). If integrator is set to Langevin, friction is 1/temperature_tau.
```

Comment:
`Time scale for thermostat.  Directly used for Berendsen integrator, or as 1/friction for Langevin integrator`